base_layer/wallet_ffi/.github/workflows/release.yml

```
@@ -0,0 +1,171 @@
+        name: sdist
+        path: base_layer/wallet_ffi/dist/*.tar.gz
+
+  test-wheels:
+    name: Test wheels
+    needs: [build-wheels, build-sdist]
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12"]
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)